app/src/components/market/sections/market_buy/market_buy.tsx

```diff
@@ -229,7 +230,7 @@ const MarketBuyWrapper: React.FC<Props> = (props: Props) => {
       const sharesAmount = formatBigNumber(displayTradedShares, baseCollateral.decimals)
       setTweet('')
       setStatus(Status.Loading)
-      setMessage(`Buying ${sharesAmount} shares...`)
+      setMessage(`Buying ${handleSmallShares(sharesAmount)} shares...`)
```

Comment:
I think it would actually be better if we just use `formatNumber` in place of `handleSmallShares` since it already does what `handleSmallShares` does and also provides a consistent system for displaying numbers

Reply:
when i try to wrap it in `formatNumber` it doesn't properly display number below 0.01 🤔

Reply:
i messed with the logic of the `formatNumber` function too, literally copy pasted in what `handleSmallShares` was doing and it still wouldn't work properly.

Reply:
Seems the issue is that `formatBigNumber` is using the default of 2 for its precision parameter, so the value being used in `formatNumber` is just `0.00`